Kafka integration (#3)
abulte/decapode: README.md

### Run

```
pip install -r requirements.txt

> adev runserver decapode/app.py

Reviewer: We're missing requirement for adev? Do we want to install `requirements_dev.txt`?

Author: I've just tested it and adev seems to be installed when we run the above

}
```

### Using Kafka integration

> The `kafka_integration` module retrieves messages with the topic `dataset` from `udata-search-service`'s Kafka instance.

Reviewer: ```suggestion
The `kafka_integration` module retrieves messages with the topic `dataset` sent by `udata`.
```

}
```

### Using Kafka integration

> The `kafka_integration` module retrieves messages with the topic `dataset` sent by `udata`.

Reviewer 2: Please update the good topics used by `kafka-integration` (not `dataset` anymore).
Please indicate in README the command to run kafka-integration `decapode run-kafka-integration`)
